### ISSUE TITLE
Unused variable

### ISSUE THREAD

**Issue by marehr** [edanor/umesimd#53]
```
/home/marehr/develope/seqan-src/include/umesimd/plugins/avx512/uint/UMESimdVecUint64_16.h: In member function ‘UME::SIMD::SIMDVec_u<long unsigned int, 16u>& UME::SIMD::SIMDVec_u<long unsigned int, 16u>::swizzlea(const UME::SIMD::SIMDSwizzle<16u>&)’:
/home/marehr/develope/seqan-src/include/umesimd/plugins/avx512/uint/UMESimdVecUint64_16.h:281:34: warning: variable ‘result’ set but not used [-Wunused-but-set-variable]
             alignas(64) uint64_t result[16];
```

Which is indeed not used:
```c++
        UME_FORCE_INLINE SIMDVec_u & swizzlea(SIMDSwizzle<16> const & sMask) {
            alignas(64) uint32_t raw_smask[16];
            alignas(64) uint64_t raw[16];
            alignas(64) uint64_t result[16];

            _mm512_store_epi32(raw_smask, sMask.mVec);
            _mm512_store_epi64(&raw[0], mVec[0]);
            _mm512_store_epi64(&raw[8], mVec[1]);

            for(unsigned int i = 0; i < 16; i++) {
                result[i] = raw[raw_smask[i]];
            }

            mVec[0] = _mm512_load_epi64(&raw[0]);
            mVec[1] = _mm512_load_epi64(&raw[1]);
            return *this;
        }
```

**Comment by edanor** [comment]
Fixed by 1263cd3e13cf79eb6678a03c1de72deb91c2fccf.

**Comment by marehr** [comment]
In the fix in 1263cd3 correct?
```c++
alignas(64) uint32_t raw_smask[16];
alignas(64) uint64_t raw[16];
alignas(64) uint64_t result[16];

_mm512_store_epi32(raw_smask, sMask.mVec);
_mm512_store_epi64(&raw[0], mVec[0]);
_mm512_store_epi64(&raw[8], mVec[1]);
// ...
mVec[0] = _mm512_load_epi64(&result[0]);
mVec[1] = _mm512_load_epi64(&result[1]);
```

The store is at position `&raw[8]`, but the load at `&result[1]`. Shouldn't it be at `&result[8]`?
